arm/compute/version.go

```diff
@@ -24,7 +24,7 @@ import (
 
 const (
 	major = "4"
-	minor = "0"
+	minor = "1"
```

Comment:
Shouldn't removal of entire service be considered as breaking change then we must increase major version.